src/Servers/Kestrel/Kestrel/test/HttpsConfigurationTests.cs

```diff
@@ -183,6 +183,29 @@ public async Task LoadEndpointCertificate(string address, bool useKestrelHttpsCo
         }
     }
 
+    [Fact]
+    public async Task CheckDefaultCertificateJustWorks()
```

Comment:
Add a config based test, and one that calls ConfigureHttpsDefaults.

Reply:
Different from https://github.com/dotnet/aspnetcore/pull/48057/files#diff-7f7197cd510d42a2d0dec7753b8a3910e5c618355620359171eb8ab02eedfacbR427 and https://github.com/dotnet/aspnetcore/pull/48057/files#diff-ce5ae844bd82fd2617175491a577eba862d04bd8490c0d0b34ac8354d1397b68R823?

Reply:
Those look good. Sorry, I hadn't seen them (old version?).